src/server/ConsoleShimPolicy.h

```diff
@@ -25,6 +25,7 @@ class ConsoleShimPolicy
 
     bool IsCmdExe() const noexcept;
     bool IsPowershellExe() const noexcept;
+    bool ApplicationMishandlesVTColors() const noexcept;

```

Comment:
The title of this function feels presumptuous and applies intent to the client application's programming. We didn't have a firm and well-developed story to introducing VT in the first place. 

I think we should choose a more neutral naming. Simply doubling down on the quirkiness of the situation like `IsVtColorQuirkRequired()` might be better.